pandora-client-web/src/components/gameContext/notificationContextProvider.tsx

```diff
@@ -23,6 +23,9 @@ export const NOTIFICATION_KEY: Readonly<Record<NotificationSource, NotificationH
 	[NotificationSource.INCOMING_FRIEND_REQUEST]: null,
 };
 
+// TODO: Make it configurable whether or not to show popup
+const MOCK_SETTING_ENABLE_POPUP = false;
```

Comment:
I do not like the idea of turning these off, I often find it really useful to get this notification when I have Pandora in another tab, this is by no means a mock feature

Disabling also removes focus that we need to actually implement this really important feature

As for how it went through review, well, it is obviously something still under development, and only a dummy text is shown, though it could have said TODO

if you want to disable it please do it with a user-configurable server side setting